content/news/2023-05-08-tpv-switch/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Job router switch on EU"
```

Comment:
```suggestion
title: "Replacing the Job Router with Zero Downtime on EU"
```

since it was done without downtime y'all should be proud of that, but on the other hand I guess it's not really mentioned how that happened, so maybe feel free to reject this change for that reason.

Reply:
Alternatively: "We Rebuilt the Engine while the Plane was Flying: Job Router switch with Zero Downtime on EU"